.github/workflows/rust.yml

```diff
@@ -31,20 +31,29 @@ jobs:
 
   wasm: 
     name: Stable - Docs / WebAssembly Build
-    runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust:
-          - stable
+        target: [ x86_64-unknown-linux-gnu, x86_64-apple-darwin ]
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+          - target: x86_64-apple-darwin
+            os: macos-latest
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout Crate
         uses: actions/checkout@v2
       - name: Checkout Toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ matrix.rust }}
+          toolchain: stable
           override: true
+      - name: Set CC and AR vars for MacOs
+        if: contains(matrix.os, 'macos')
+        run: |
+          echo "CC=/usr/local/opt/llvm/bin/clang" >> $GITHUB_ENV
```

Comment:
I don't think this will work because the CC env variable is overriden  in `contrib/test.sh`?

Reply:
Interesting. It works though: https://github.com/bonomat/rust-secp256k1/pull/1/checks?check_run_id=3075037685

Reply:
Unfortunately, it doesn't if you actually check the log output: https://github.com/bonomat/rust-secp256k1/pull/1/checks?check_run_id=3075037685#step:6:853

For some reason, the script is not failing the build. @rishflab also ran into this problem.

Reply:
Thanks for pointing this out.